src/main/java/com/mineboundteam/minebound/magic/DefensiveSpells/ElectricDefensiveSpell.java

```diff
@@ -0,0 +1,149 @@
+import net.minecraftforge.common.ForgeConfigSpec;
+import net.minecraftforge.event.entity.living.LivingDamageEvent;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.fml.event.config.ModConfigEvent;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.List;
+
+@Mod.EventBusSubscriber(modid = MineBound.MOD_ID)
+public class ElectricDefensiveSpell extends ActiveSpellItem {
+    public static final String ACTIVE_TAG = "minebound.electric_defensive.active";
+
+    private final int initialManaCost;
+    private final int manaCostPerHit;
+    private final int durationInTicks;
+
+    public ElectricDefensiveSpell(Properties properties, ElectricDefensiveSpellConfig config) {
+        super(properties, config.LEVEL, MagicType.ELECTRIC, SpellType.DEFENSIVE);
+
+        this.initialManaCost = config.INITIAL_MANA_COST.get();
+        this.manaCostPerHit = config.MANA_COST_PER_HIT.get();
+        this.durationInTicks = config.DURATION_TICKS.get();
+    }
+
+    @Override
+    public void use(ItemStack stack, InteractionHand usedHand, Level level, Player player) {
+        stack.getOrCreateTag().putBoolean(ACTIVE_TAG, true);
+        reduceMana(initialManaCost, player);
+    }
+
+    @Override
+    public void onUsingTick(ItemStack stack, InteractionHand usedHand, Level level, Player player, int tickCount) {
+    }
+
+    @Override
+    public void releaseUsing(ItemStack stack, InteractionHand usedHand, Level level, Player player) {
+        stack.getOrCreateTag().putBoolean(ACTIVE_TAG, false);
+    }
+
+    @SubscribeEvent
+    public static void triggerSpell(LivingDamageEvent event) {
+        if (event.getEntityLiving() instanceof ServerPlayer player && event.getSource().getEntity() instanceof LivingEntity entity) {
+            boolean spellTriggered = triggerSpell(player, entity, getSelectedSpell(player, PlayerSelectedSpellsProvider.PRIMARY_SPELL));
+            if (!spellTriggered) {
+                spellTriggered = triggerSpell(player, entity, getSelectedSpell(player, PlayerSelectedSpellsProvider.SECONDARY_SPELL));
+            }
+            if (!spellTriggered) {
+                triggerSpell(player, entity, player.getUseItem());
+            }
+        }
+    }
+
+    protected static boolean triggerSpell(Player player, LivingEntity entity, ItemStack selectedSpell) {
+        if (selectedSpell.getItem() instanceof ElectricDefensiveSpell spell) {
+            if (selectedSpell.getOrCreateTag().getBoolean(ACTIVE_TAG)) {
+                entity.addEffect(new MobEffectInstance(EffectRegistry.ELECTRIC_DEBUFF.get(), spell.durationInTicks, spell.level.getValue()));
+                reduceMana(spell.manaCostPerHit, player);
+                return true;
+            }
+        }
+        return false;
+    }
+
+    @Override
+    public void appendHoverText(ItemStack pStack, @Nullable Level pLevel, List<Component> pTooltipComponents, TooltipFlag pIsAdvanced) {
```

Comment:
Should wait for Tooltip changes, or tooltip changes should wait for this

Reply:
#174 will probably wait for this and #173.